c7n/resources/rdscluster.py

```diff
@@ -399,6 +399,70 @@ def process(self, clusters):
                 client.exceptions.InvalidDBClusterStateFault)
 
 
+@RDSCluster.action_registry.register('modify-db-cluster')
+class ModifyDbCluster(BaseAction):
+    """Modifies an RDS instance based on specified parameter
+    using ModifyDbInstance.
+
+    'Update' is an array with with key value pairs that should be set to
+    the property and value you wish to modify.
+    'Immediate" determines whether the modification is applied immediately
+    or not. If 'immediate' is not specified, default is false.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: disable-db-cluster-deletion-protection
+                resource: rds-cluster
+                filters:
+                  - DeletionProtection: true
+                  - PubliclyAccessible: true
+                actions:
+                  - type: modify-db-cluster
+                    update:
+                      - property: 'DeletionProtection'
+                        value: false
+                    immediate: true
+    """
+
+    schema = type_schema(
+        'modify-db-cluster',
+        immediate={"type": 'boolean'},
+        update={
+            'type': 'array',
+            'items': {
+                'type': 'object',
```

Comment:
this should be a passthrough on parameters with shape validation instead of hardcoding a limited set of things. see cloudtrail resource update, or ssm send command for examples.